tools/test-tools/src/assignTestPorts.ts

```diff
@@ -40,8 +45,8 @@ export function getPackageInfo(): PackageInfo[] {
 	}
 }
 
-export function writePortMapFile(initialPort: number): void {
-	const info: PackageInfo[] = getPackageInfo();
+export function writePortMapFile(initialPort: number, packageManager: 'pnpm' | 'lerna'): void {
```

Comment:
Looks like passing the new argument is missing in assign-test-ports script - line 38